### PR TITLE
36 qrcode class

### PR DESCRIPTION
**Pull request by KrishLal**

### CODE REVIEW

**Comment by manan3172003** [comment]
Do not merge yet the MainActivity.java has been edited

**Comment by KrishLal** [comment]
Yes, we'll need to manually review the changes and then merge carefully

File: code/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
         android:theme="@style/Theme.Lordofthewings"
         tools:targetApi="31">
         <activity
-            android:name=".Views.MainActivity"
```
Comment (mansooranis) [inline code/app/src/main/AndroidManifest.xml:16]: Add Views.MainActivity

**Comment by KrishLal** [comment]
im still testing reverts to go back to a certain commit to only add the files required. don't merge still

**Comment by mansooranis** [comment]
@KrishLal any updates on this pull request?

**Pull request closed without merging**